.github/workflows/release-from-main.yml

```
@@ -0,0 +1,91 @@
+name: Release iOS Demo App
+
+on:
+  push:
+    branches:
+      - main
```

```suggestion
  push:
    branches:
      - main
    paths:
      - VERSION
```

Other repos limit releases to only when the VERSION file in the root changes. This makes it so we don't release when unrelated things change like Github actions dependency versions or similar. Main is still in a releasable state but it's just not worth making that extra version